release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -65,6 +65,20 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-release-notes-edge-computing_{context}"]
 === Edge computing
 
+[id="ocp-4-18-edge-computing-sno-shutdown_{context}"]
+==== Shutting down and restarting single-node clusters up to 1 year after cluster installation
+
+With this release, you can shut down and restart single-node clusters up until 1 year after cluster installation. If certificates expired while the cluster was shut down, you must approve certificate signing requests (CSRs) upon restarting the cluster.
+
+Before this update, you could shut down and restart single-node clusters for only 120 days after cluster installation.
+
+[IMPORTANT]
+====
+It is supported to shut down single-node clusters only if there are no workloads running on the cluster.
```

Comment:
```suggestion
Evacuate all workload pods from the {sno} cluster before you shut it down.
```

Reply:
@vrutkovs Is this change okay to make? Is it okay if they had workloads on it but evacuated them, or do we expect them to shut down before workload were ever run on them?

Reply:
Yes, good idea

Reply:
Perfect thanks, updated!